docs/spec/v1beta3/providers.md

```diff
@@ -945,6 +948,32 @@ metadata:
 stringData:
     address: https://username:password@<alertmanager-url>/api/v2/alerts/"
 ```
+Bearer Token Authentication:
+Create a Secret with [the `token`](#token-example), and a `alertmanager` Provider with a [Secret
```

Comment:
```suggestion
Create a Secret with [the `token`](#token-example), and an `alertmanager` Provider with a [Secret
```

If the word following an indefinite article starts with the sound of a vowel, like "alert", the article is `an`. :grin: 

It's not part of your change but please fix this on the basic auth docs right above as well :pray: Much appreciated :grin: